Sources/TensorFlow/Operators/Dataset.swift

```diff
@@ -62,7 +62,7 @@ public extension Dataset {
 }
 
 public extension Dataset {
-    /// Creates a dataset from a batch of elements as a tensor.
+    /// Returns a tensor by creating a dataset from a batch of elements.
```

Comment:
The wording described in #160 only applies to non-mutating instance methods, not initializers. The original doc comment here is already okay: All initializer doc comments should start with `Creates a ...`.

Reply:
Yes, Sorry. I wasn't sure of this change. I'll change it.